src/components/miningtutorial/TecSpecs.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import CloseTutorialButton from './CloseTutorialButton';
+import MiningTutorialHeadline from './MiningTutorialHeadline';
+import MiningTutorialText from './MiningTutorialText';
+
+export default function TecSpecs({ hideTutorialArticle }) {
+  return (
+    <TutorialArticleItem>
+      <MiningTutorialHeadline headline={'Technische Anforderungen'} />
+      <MiningTutorialText
+        text={
+          'Für dieses Tutorial gehen wir von einer weit verbreiteten Hard- und Softwarekombination aus, die für die meisten Otto-Normalverbraucher relevant und für das Mining selbst auch mit am Profitabelsten ist:'
+        }
+      />
+      <TechSpecList>
+        <TechSpecListItem>Desktop-PC</TechSpecListItem>
```

Comment:
Do you need a key prop here? you could use Math.random() * Math.random() ore nanoid()